inc/healthcheck/inc/cavalcade/namespace.php

```diff
@@ -15,6 +15,7 @@
  */
 function bootstrap() {
 	add_action( JOB_HOOK, __NAMESPACE__ . '\\set_last_run' );
+	// @codingStandardsIgnoreLine
```

Comment:
!

Reply:
It was complaining about modifying the cron array. Maybe it's a new sniff